src/actions/socialLogin/FacebookActions.js

```diff
@@ -44,9 +44,11 @@ const LoginFacebook = token => (dispatch) => {
     .then((resp) => {
       sessionStorage.setItem('token', resp.data.user.token);
       sessionStorage.setItem('user_email', resp.data.user.email);
+      sessionStorage.setItem('logged_in', true);
```

Comment:
Since you have removed the success page I suggest storing of user_email in sessionStorage should be removed too because it no longer has use.